src/domain/core/src/entities/current_account_config.rs

```diff
@@ -0,0 +1,36 @@
+// Copyright Kamu Data, Inc. and contributors. All rights reserved.
+//
+// Use of this software is governed by the Business Source License
+// included in the LICENSE file.
+//
+// As of the Change Date specified in that file, in accordance with
+// the Business Source License, use of this software will be governed
+// by the Apache License, Version 2.0.
+
+use opendatafabric::AccountName;
+
+/////////////////////////////////////////////////////////////////////////////////////////
+
+#[derive(Debug, Clone)]
+pub struct CurrentAccountConfig {
```

Comment:
When we integrate authorization library this will likely be called `Subject`, perhaps we should call it that from the beginning?

Reply:
Not a big deal for now - let's leave it until we integrate authorization

Reply:
I renamed, but I still don't understand the convention you are referring too.